dist/config.d/10-identity.toml

```diff
@@ -0,0 +1,15 @@
+# Configure agent identity.
+[identity]
+
+# Node group, used for cluster-wide reboot orchestration (e.g. Airlock).
+group = "default"
+
+# Node ID, in sd-id128(3) format.
+# By default it is automatically computed as
+# `systemd-id128 machine-id -a de35106b6ec24688b63afddaa156679b`
```

Comment:
Not new here, though what's the rationale for doing `-a <app-id>` vs. just using the machine-id directly?

Reply:
Because we need to send the uuid over the network (to Cincinnati backend), and the machine-id is not supposed to be leaked that way. See https://www.freedesktop.org/software/systemd/man/machine-id.html:

> This ID uniquely identifies the host. It should be considered "confidential", and must not be exposed in untrusted environments, in particular on the network. If a stable unique identifier that is tied to the machine is needed for some application, the machine ID or any part of it must not be used directly. Instead the machine ID should be hashed with a cryptographic, keyed hash function, using a fixed, application-specific key. That way the ID will be properly unique, and derived in a constant way from the machine ID but there will be no way to retrieve the original machine ID from the application-specific one. The sd_id128_get_machine_app_specific(3) API provides an implementation of such an algorithm.